query/src/servers/http/clickhouse_handler.rs

```diff
@@ -164,6 +178,15 @@ pub async fn clickhouse_handler_post(
         .await
         .map_err(InternalServerError)?;
 
+    let settings = session.get_settings();
```

Comment:
what about extract the dup code into a funtion.

Reply:
make sense